code/modules/reagents/reagent_containers/food/snacks/pizza.dm

```diff
@@ -82,7 +82,7 @@
 
 /obj/item/reagent_containers/food/snacks/sliceable/pizza/vegetablepizza
 	name = "vegetable pizza"
-	desc = "No Tomato Sapiens were harmed during the baking of this pizza."
+	desc = "A pizza made with vegatables to replace cheese."
```

Comment:
A vegetable pizza does not replace the cheese, it replaces the meat.

"A pizza made with vegetables to replace meat."